paldo-riddles/package.json

```diff
@@ -11,36 +11,36 @@
   },
   "scripts": {
     "start": "node server",
-    "test": "lab -a code -I '__core-js_shared__' -L",
+    "test": "lab -a @hapi/code -I 'core,__core-js_shared__,CSS' -L",
```

Comment:
I'm a little surprised by this!  I sort of thought that our base objection setup didn't have any globals anymore.  I'll have to give this a whirl with a fresh install some time.

Reply:
ohhh boi. yea, I forget exactly how this showed up or why i thought it was specific to `hapi-swagger`....damn, yea, sorry! this update was effectively punting on that instead of figuring out why it was happening / if it were an issue, IIRC